x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/MachineLearning.java

```diff
@@ -887,101 +881,6 @@ protected Clock getClock() {
     public UnaryOperator<Map<String, IndexTemplateMetaData>> getIndexTemplateMetaDataUpgrader() {
         return templates -> {
```

Comment:
I have not replaced this with returning an operator that does nothing yet because it is unclear whether `IndexTemplateRegistry` fully addresses upgrading issues. I'm waiting to hear on this, until then the PR is WIP.